libvirt-wrapper.py

```diff
@@ -0,0 +1,91 @@
+import libvirt
+import sys
+import argparse
+
+class LibvirtConn:
+    (ECONN, EFIND, ECREATE, EDELETE, ESTART, ESTOP, EREBOOT) = (1,2,3,4,5,6,7)
+    def __init__(self):
+        try:
+            self.conn = libvirt.open("qemu:///system")
+        except libvirt.libvirtError:
+            raise SystemExit(self.ECONN)
+
+    def find_vm_dec(func):
+        def closure(self, name, *args, **kwargs):
+                try:
+                    vm = self.conn.lookupByName(name)
+                except libvirt.libvirtError:
+                    raise SystemExit(self.EFIND)
```

Comment:
My previous note on SystemExit applies here, as well as for other methods in this class.